lib/controllers/rsvps.dart

```diff
@@ -134,6 +142,13 @@ class InstanceRsvpsController
           savedRsvp));
     }
 
+    if (savedRsvp != null) {
```

Comment:
savedRsvp will also be null if a user deletes their RSVP after having created in—in which case we should find and delete the event from their calendar